terraform/state-persistence.tf

```diff
@@ -0,0 +1,9 @@
+data "terraform_remote_state" "gt-site-cluster" {
```

Comment:
Odd, I've noticed it pushing and pulling remote state via normal `terraform apply` and `terraform destroy`.